scripts/utils/DonationUsdValueUtility.js

```diff
@@ -40,9 +40,9 @@ class DonationUsdValueUtility {
 
   async setDonationUsdValue(donation) {
     const { createdAt, token, amount } = donation;
-    const { symbol } = token;
```

Comment:
Sometimes tokens maybe null  so destructuring throw exception, and because of that this line moved to try-catch scope